app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontSetProductRatingStarsActionGroup.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontSetProductRatingStarsActionGroup">
```

Comment:
I would suggest moving this action group into the `Review` module instead of `Catalog`

Reply:
Has been moved.